### PR TITLE
change wordings mainly from AWS landing zone to OCIO Landing zone

### PR DESCRIPTION
**Pull request by abibat-adesina**

### CODE REVIEW

File: docs/bc-govs-aws-landing-zone-overview.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
 
 5. **Long-term management and evolution**: This tool not only helps deploy applications initially but also aids in their ongoing management and evolution. This feature is crucial for government applications, which frequently need updates and adjustments to align with changing policy requirements and citizen needs
 
-6. **Leveraging AWS capabilities**: When ministry teams build on AWS, they can tap into a wide range of services and capabilities, spanning from advanced analytics to AI and machine learning tools. This integration has the potential to greatly boost the functionality and reach of government applications
+6. **Leveraging cutting-egde capabilities in public cloud**: When ministry teams build in public cloud, they can tap into a wide range of services and capabilities, spanning from advanced analytics to AI and machine learning tools. This integration has the potential to greatly boost the functionality and reach of government applications
```
Comment (Pilargit12) [inline docs/bc-govs-aws-landing-zone-overview.md:25]:
```suggestion
6. **Leveraging cutting-egde capabilities in the public cloud**: When ministry teams build in public cloud, they can tap into a wide range of services and capabilities, spanning from advanced analytics to AI and machine learning tools. This integration has the potential to greatly boost the functionality and reach of government applications
```

File: docs/bc-govs-aws-landing-zone-overview.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
 
-For B.C. government ministry teams developing applications in the AWS Public Cloud, the ASEA provides a secure, compliant, and efficient pathway. This facilitates the creation of innovative and responsive applications that effectively serve the public.
+For B.C. government ministry teams developing applications in the AWS Public Cloud, the OCIOlanding zone provides a secure, compliant, and efficient pathway. This facilitates the creation of innovative and responsive applications that effectively serve the public.
```
Comment (Pilargit12) [inline docs/bc-govs-aws-landing-zone-overview.md:27]:
```suggestion
For B.C. government ministry teams developing applications in the AWS Public Cloud, the OCIO landing zone provides a secure, compliant, and efficient pathway. This facilitates the creation of innovative and responsive applications that effectively serve the public.
```

**Review by Pilargit12 — CHANGES_REQUESTED** [review]
Looks good to go

**Review by abibat-adesina — COMMENTED** [review]
Thank you

**Pull request merged**